spec/index.bs

```diff
@@ -8684,10 +8698,15 @@ dictionary GPUCanvasConfiguration {
 };
 </script>
 
-Issue: For SDR, it doesn't matter whether the default colorSpace of "srgb" means extended-srgb
-or clamped-srgb. However, when we add HDR canvas output (representing pixel values > 1),
-we need to choose which of those two is the default. Currently the upstream specs haven't worked
-out this question just yet.
+### Canvas Color Space ### {#canvas-color-space}
+
+The chrominance of color values outside of the [0, 1] range is not to be clamped; extended values
+may be used to display colors outside of the gamut defined by the canvas' color space's primaries
+(configured {{GPUCanvasConfiguration/format}} permitting).
+This is in contrast with luminance, which is to be clamped to the maximum standard dynamic range
+luminance.
+
+Issue: ... unless high dynamic range is explicitly enabled for the canvas element.
```

Comment:
I'm not 100% sure this is perfect, but we can implement and revisit if it ends up needing changes.

Reply:
Added link to where I got this wording from
https://github.com/WICG/canvas-color-space/blob/6bc492d2564abec316e2c5663aec9f9fa523a6a2/CanvasColorSpaceProposal.md#compositing-of-the-htmlcanvaselement